account_tax_balance/models/account_move.py

```diff
@@ -24,22 +24,28 @@ class AccountMove(models.Model):
         'line_ids.account_id.user_type_id.type'
     )
     def _compute_move_type(self):
-        def _balance_get(line_ids, internal_type):
-            return sum(line_ids.filtered(
-                lambda x: x.account_id.internal_type == internal_type).mapped(
-                    'balance'))
+        refund_types = ('in_refund', 'out_refund')
+
+        def _is_refund(line_ids, internal_type):
+            """Check whether all the lines of type `internal_type`
+            come from a refund."""
+            line_ids = line_ids.filtered(
+                lambda x: x.account_id.internal_type == internal_type)
+            line_types = line_ids.mapped('invoice_id.type')
```

Comment:
The problem here is those moves without invoice_id.

Reply:
I have searched the code and found no better way to determine if 'something' is a refund.
Using the balance has the issues described in this PR that I'd summarize in: what is **0**?

The moves without `invoice_id` would be represented as `Payable`/`Receivable`, according to their account type.
That doesn't seem that bad to me because if there is no refund (that is, an invoice of type `*_refund`), why should any move be identified as a refund?

This said, I'm no functional so if there is some better way to recognize a refund `account.move` please, I'm listening :)

Reply:
@SimoRubi according to https://github.com/OCA/account-financial-reporting/pull/732/files#r502467979 could you fall back the previous behaviour in case of moves without `invoice_id`?

Reply:
Added fallback to previous behavior

Reply:
thanks!

Reply:
@pedrobaeza can you update your review?
Thanks!